### ISSUE TITLE
force header case for a single header

### ISSUE THREAD

**Issue by yovanoc** [apify/got-scraping#134]
is there a way to override per-request header casing (e.g choosing a uppercase for a single header even when http2) ?

**Comment by B4nan** [comment]
i think the headers are normalized directly in `got` (to lower case), sounds like a wontfix to me.

**Comment by B4nan** [comment]
Yes but even if we hadn't done that, it would have happened on the lower level, as got does the same

https://github.com/sindresorhus/got/issues/1335
